src/sentry/interfaces/breadcrumbs.py

```diff
@@ -83,7 +70,10 @@ def to_json(self):
     @classmethod
     def normalize_crumb(cls, crumb):
         ty = crumb.get('type') or 'default'
-        level = crumb.get('level') or 'info'
+        level = crumb.get('level')
+        if level not in LOG_LEVELS_MAP or level == 'log':
```

Comment:
Isn't `log` not in `LOG_LEVELS_MAP` anyways?

Reply:
No, it only contains the standard names: sample, debug, info, warning, error, fatal

Reply:
Then the `or` can go.

Reply:
fixed